src/pages/about-us.js

```diff
@@ -1,10 +1,10 @@
 import Head from 'components/Head'
-import getAboutUsData from '../../data/about-us'
+import aboutUsData from '../../data/about-us'
```

Comment:
another way is like this:

```suggestion
import { meta, hero } from '../../data/about-us'
```

But maybe you prefer to know where the variables are coming from lower in the code

Reply:
Yes absolutely, I'll keep it this way but thanks anyway for the suggestion ! 😊